app/helpers/items_helper.rb

```diff
@@ -1,10 +1,18 @@
 # Encapsulates methods that need some business logic
 module ItemsHelper
-  def dollar_value(value, addition = '')
+  def dollar_presentation(value)
+    dollars = cents_to_dollar value
```

Comment:
I think the pattern in the app is to use parentheses when we are passing in a value.

Reply:
Added/restored

Reply:
In case you want to enforce (& auto-correct?) this with Rubocop, enabling `Style/MethodCallWithArgsParentheses`
produced 3,728 offenses (all auto-correctable)

Adding
```
Style/MethodCallWithArgsParentheses:
  Exclude:
    - 'app/views/**/*.json.jbuilder'
    - 'config/application.rb'
    - 'config/routes.rb'
    - 'db/partners_migrate/*.rb'
    - 'spec/**/*.rb'
```

to `.rubocop_todo.yml` reduced that to 357 offenses, but many still appear idiomatic to me—e.g.,

```
raise ActiveRecord::Rollback
```

```
super params
```

```
render action: :edit
```

excluding `app/controllers/**./*.rb`, too, reduces it to 120 offenses

🤷‍♂️